src/components/time/time.test.tsx

```diff
@@ -131,24 +133,22 @@ describe("Time component", () => {
       />
     );
 
-    const user = userEvent.setup();
-    await act(async () => {
-      await user.tab();
+    const user = userEvent.setup({
+      advanceTimers: jest.advanceTimersByTime,
+      delay: null,
```

Comment:
We should be cautious about using the `delay` option.

By default, RTL adds a small delay between subsequent inputs, like key presses, to simulate how a real user would provide input. Setting `delay` to `null` removes this delay entirely, so it could potentially mask potential issues related to timeouts or promises in our code.